src/main/resources/templates/printService/caseDetailsSOL.html

```diff
@@ -39,7 +37,13 @@
 </tr>
 {% endmacro %}
 {% macro notApplyingReason(reason) %}
-<td valign="top">{% if reason == "DiedBefore" %}They died before the deceased.{% endif %}{% if reason == "DiedAfter" %}They died after the deceased.{% endif %}{% if reason == "PowerReserved" %}They're holding power reserved.{% endif %}{% if reason == "Renunciation" %}They have renounced.{% endif %}</td>
+<td valign="top">{% if reason == "DiedBefore" %}They died before the deceased.{% endif %}
+    {% if reason == "DiedAfter" %}They died after the deceased.{% endif %}
+    {% if reason == "PowerReserved" %}They're holding power reserved.{% endif %}
+    {% if reason == "Renunciation" %}They have renounced.{% endif %}
+    {% if reason == "PowerOfAttorney" %}They have appointed or wish to appoint another person to act as their attorney.{% endif %}
+    {% if reason == "MentallyIncapable" %}They lack capacity to act as executor and the will allows for a substitution.{% endif %}
+</td>
```

Comment:
we spoke briefly about this - i think that changing it is probably good for readability but i wonder if this should be:
```suggestion
<td valign="top">{% if reason == "DiedBefore" %}They died before the deceased.
    {% elseif reason == "DiedAfter" %}They died after the deceased.
    {% elseif reason == "PowerReserved" %}They're holding power reserved.
    {% elseif reason == "Renunciation" %}They have renounced.
    {% elseif reason == "PowerOfAttorney" %}They have appointed or wish to appoint another person to act as their attorney.
    {% elseif reason == "MentallyIncapable" %}They lack capacity to act as executor and the will allows for a substitution.
    {% else %}NO TEXT FOR REASON {{ reason }}
    {% endif %}
</td>
```